kuma/payments/tests/test_views.py

```diff
@@ -163,3 +166,21 @@ def test_template_render_not_logged_in_recurring_payment_initial_view(mock_enabl
     assert '<input type="email"' in response.content
     assert 'id="id_donation_amount"' in response.content
     assert 'id="id_donation_choices"' in response.content
+
+
+@pytest.mark.django_db
+@mock.patch('kuma.payments.views.enabled')
+def test_template_render_not_logged_in_payment_view(mock_enabled, client, settings):
+    """If enabled, contribution error page is returned."""
```

Comment:
+1 for a docstring, but -2 for a mismatch between the string and the test

Reply:
I've addressed this and other issues in https://github.com/mozilla/kuma/pull/5084